CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2021, 鍾淯丞, 周杰仕, 林仁鴻. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+cmake_minimum_required(VERSION 3.20)
+project(TestRequestParser)
+enable_testing()
+
+set (CMAKE_CXX_STANDARD 14)
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/cmake/Modules/")
+
+find_package(GTest REQUIRED)
+
+include_directories(${GMOCK_ROOT}/include)
+include_directories(${GMOCK_ROOT}/gtest/include)
+include_directories(${GTEST_INCLUDE_DIRS})
+
+include_directories(./src/Header)
```

Comment:
Avoid using relative path, 
use CMAKE_SOURCE_DIR, or PROJECT_SOURCE_DIR, or CMAKE_CURRENT_SOURCE_DIR instead.

Reply:
This is a reasonable code review.

Reply:
Done